### PR TITLE
Updating Readme according to #1082

### PR DESCRIPTION
**Pull request by ksaadDE**
Updating Readme according to #1082

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,12 +95,7 @@ This handy table lists all the methods you can use to install Spotube:
   </tr>
   <tr>
     <td>AppImage</td>
-    <td>
-      <a href="https://github.com/KRTirtho/spotube/releases/latest/download/Spotube-linux-x86_64.AppImage">
-        <img width="220" alt="Download AppImage" src="https://user-images.githubusercontent.com/61944859/169455015-13385466-8901-48fe-ba90-b62d58b0be64.png">
-      </a>
-      <p><b>Note:</b> <a href="https://github.com/TheAssassin/AppImageLauncher">AppimageLauncher</a> is required!</p>
-    </td>
+    <td>Currently not available see https://github.com/KRTirtho/spotube/issues/1082</td>
```
[the remark this answers is not in the document]
Comment (ksaadDE) [reply README.md]: I updated the file

**Review by ksaadDE — COMMENTED** [review]
Improved according to suggestion.

**Review by OpenCode — APPROVED** [review]

**Review by varshith257 — APPROVED** [review]
LGTM

**Comment by OpenCode** [comment]
@KRTirtho Can you merge this PR?

**Review by KRTirtho — APPROVED** [review]

**Pull request merged**